assets/js/Model.js

```diff
@@ -42,3 +106,55 @@ App.Model.prototype.getSourceTranslation = function ( nodeId ) {
 		exists: true
 	};
 };
+
+App.Model.prototype.getTargetTranslation = function ( nodeId ) {
+	if ( this.translations[ nodeId ][ this.targetLang ] === undefined ) {
+		return '';
+	}
+	return this.translations[ nodeId ][ this.targetLang ].text;
+};
+
+/**
+ * Load model values from LocalStorage.
+ */
+App.Model.prototype.loadFromLocalStorage = function () {
+	this.setSourceLang( this.getLocalStorage( 'sourceLang' ) );
+	this.setTargetLang( this.getLocalStorage( 'targetLang' ) );
+};
+
+/**
+ * Get a value from our LocalStorage item.
+ * @protected
+ * @param {string} key
+ * @return {*}
+ */
+App.Model.prototype.getLocalStorage = function ( key ) {
```

Comment:
Super nitpicky here, but I'd rename the method to "getLocalStorageValue" since it's not giving back the whole of the local storage, but only a single key value.

Reply:
Done.